eoscompanion/main.py

```diff
@@ -149,17 +149,79 @@ def do_dbus_register(self, connection, object_path):  # pylint: disable=argument
 
     def do_dbus_unregister(self, connection, object_path):  # pylint: disable=arguments-differ
         '''Invoked when we lose a D-Bus connection.'''
-        log('Lost session d-bus connection at {path}'.format(path=object_path))
+        logging.warning('Lost session d-bus connection at %s', object_path)
         return Gio.Application.do_dbus_unregister(self,
                                                   connection,
                                                   object_path)
 
     def do_activate(self):  # pylint: disable=arguments-differ
         '''Invoked when the application is activated.'''
-        log('Activated')
+        logging.info('Activated')
         return Gio.Application.do_activate(self)
 
 
+COMPANION_APP_CONFIG_FILES = [
+    '/var/run/host/usr/share/eos-companion-app/config.ini',
```

Comment:
Is the order correct here? I have adjusted my config file in "/etc/eos-companion-app/config.ini" but "/usr/share/eos-companion-app/config.ini" takes precedence. That file owned by root is not writeable for me - while the config under /etc is for me as it has been created by the companion-app-helper.

If we keep this order maybe, we should check as well if the file has the 'loglevel' set an not bail out when the file is available.

Reply:
You're right - the order should go the other way around

Reply:
Fixed.